### PR TITLE
release: 2.63.1

### PR DESCRIPTION
**Pull request by ernestl**
This release is a minimum change to improve steam performance and nvidia support for steam. 
It is a deb only release targeting Noble 24.04.1.

**Outstanding content:** 
 - [Backport of AppArmor allow all](https://github.com/canonical/snapd/pull/14307)
 - Related integration tests

TODO Generate changelogs with:
DEBEMAIL="Ernest Lotter <ernest.lotter@canonical.com>" release-tools/changelog.py 2.63.1 2061179 NEWS.md

https://bugs.launchpad.net/ubuntu/+source/snapd/+bug/2061179

### CODE REVIEW

**Review by zyga — COMMENTED** [review]
I have this side too but I was about to open one with apparmor as well

File: spread.yaml [hunk 1]
```diff
@@ -126,6 +126,9 @@ backends:
             - ubuntu-20.04-64:
                   storage: 12G
                   workers: 8
+            - ubuntu-24.10-64:
+                  storage: 12G
+                  workers: 8
```
Comment (pedronis) [inline spread.yaml:131]: a nitpick for master, not here: but shouldn't these backends be ordered somehow?

**Review by pedronis — APPROVED** [review]
thanks, noticed something for master

**Comment by ernestl** [comment]
This proposal was based on a minimum change that did not introduce apparmor min version concept for mqueue and allow all. 
After discussion with Zyga we decided to rather go with his version that did introduce the min version check for apparmor features.

Closing this in favour of https://github.com/canonical/snapd/pull/14311.

**Comment by ernestl** [comment]
This was duplicated, went with the other PR: https://github.com/canonical/snapd/pull/14311

**Pull request closed without merging**